src/exercises/exerciseTypes/wordInContextExercises/WordInContextExercise.js

```diff
@@ -69,8 +69,13 @@ export default function WordInContextExercise({
 
   function equalAfterRemovingSpecialCharacters(a, b) {
     // from: https://stackoverflow.com/a/4328546
-    let first = a.replace(/[^\w\s']|_/g, "").replace(/\s+/g, " ");
-    let second = b.replace(/[^\w\s']|_/g, "").replace(/\s+/g, " ");
+    // TR (28-03-2025) We need to support cases where the word might be a: word', b:word
```

Comment:
sorry, i can't parse the cases? the first is word followed by ', e.g. `childrens'` ? and the second is when the tokenizer included `:` in the word?

Reply:
ah, nevermind, i see that `a` and `b` are function parameters.